src/otaclient/errors.py

```diff
@@ -41,6 +41,8 @@ class OTAErrorCode(int, Enum):
     E_INVALID_STATUS_FOR_OTAROLLBACK = 202
     E_OTA_IMAGE_INVALID = 203
     E_UPDATE_REQUEST_COOKIE_INVALID = 204
+    E_CLIENT_UPDATE_SAME_VERSIONS = 205
+    E_CLIENT_UPDATE_FAILED = 206
```

Comment:
new error codes to handle
- Same Version in Client Update
- Download/Mount Errors in Client Update